client/src/endpoint/ledger.rs

```diff
@@ -126,6 +126,49 @@ impl EndPoint for All {
     }
 }
 
+/// Represents the ledger details endpoint for the stellar horizon server. The endpoint
+/// will return a single ledger's details.
+///
+/// https://www.stellar.org/developers/horizon/reference/endpoints/ledgers-single.html
+///
+/// ## Example
+/// ```
+/// use stellar_client::sync::Client;
+/// use stellar_client::endpoint::ledger;
+///
+/// let client      = Client::horizon_test().unwrap();
+/// let endpoint    = ledger::Details::new(12345);
+/// let record      = client.request(endpoint).unwrap();
+/// #
+/// # assert!(record.sequence() == 12345); //
+/// ```
+#[derive(Debug, Default)]
+pub struct Details {
+    sequence: u32,
+}
+
+impl Details {
+    /// Returns a new endpoint for ledger details. Hand this to the client in order to request
+    /// details about a ledger.
+    ///
+    /// In Stellar, the sequence number is the equivalent of Bitcoin's block height. Thus, by 
```

Comment:
@correlator added a note here about the sequence number. Is this sort of what you had in mind?

Reply:
Love it